### PR TITLE
[localization] add info to LocProject.json

### PR DESCRIPTION
**Pull request by strega-nil-ms**
important for localization stuff.

### CODE REVIEW

**Review by ras0219-msft — APPROVED** [review]
Not sure what the fields mean, but otherwise LGTM.

File: LocProject.json [hunk 1]
```diff
@@ -1 +1,14 @@
-{}
+{ 
+  "Projects": [
+    {
+      "LanguageSet": "",
+      "LocItems": [
+        {
+          "SourceFile": "locales/en.json",
+          "CopyOption": "LangIDOnName",
```
[the remark this answers is not in the document]
Comment (strega-nil-ms) [reply LocProject.json:8]: What are lcl files?

File: LocProject.json [hunk 2]
```diff
@@ -1 +1,14 @@
-{}
+{ 
+  "Projects": [
+    {
+      "LanguageSet": "",
+      "LocItems": [
+        {
+          "SourceFile": "locales/en.json",
```
[the remark this answers is not in the document]
Comment (strega-nil-ms) [reply LocProject.json:7]: mmh, yeah sure

[the remark this answers is not in the document]
Comment (strega-nil-ms) [reply LocProject.json:7]: does this still result in `locales/strings.en.json` being generated by the localization team, or should we add both?

File: LocProject.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (cristianosuzuki77) [reply LocProject.json:8]: See in https://aka.ms/allaboutloc 'Step by Step' at the top of the page.

[the remark this answers is not in the document]
Comment (strega-nil-ms) [reply LocProject.json:8]: @cristianosuzuki77 like that?

**Pull request merged**